app/helpers/application_formatters_helper.rb

```diff
@@ -24,8 +24,8 @@ def display_user(user)
   #
   # @param [User] user The user to display
   # @return [String] A url for the image.
-  def user_image(user)
-    image_path(user.profile_photo.medium.url) if user&.profile_photo&.medium&.url
+  def user_image(user, url: false)
```

Comment:
nit: `fullUrl`
Also, add the param in the docstring above

Reply:
What do you mean by `fullUrl`?

I used `url` because it's precisely defined by Rails. `path` means without the hostname, while `url` means with hostname, hence the helpers `*_path` and `*_url`.